### ISSUE TITLE
ground rendering failure, 3D mode, close zooms

### ISSUE THREAD

**Issue by boocha** [organicmaps/organicmaps#1379]
Ground rendering failure on a close zooms with 3D-buildings mode enabled. All surfaces except buildings become dark after zooming in and when navigating. 

Night mode ok. Day mode without 3D ok.

[https://www.youtube.com/watch?v=d2kwbUyX_-I](url)

Specification:
RockChip PX5 rk3368
PowerVR G6110 GPU
Support OpenGL ES 1.1/2.0/3.1, OpenCL, DirectX9.3

**Comment by vng** [comment]
@boocha Could you please check the latest app update, if it is still relevant?

**Comment by boocha** [comment]
> @boocha Could you please check the latest app update, if it is still relevant?
I checked the latest version of the app. Same bug. Without changes.

**Comment by biodranik** [comment]
What version of Android is it?

@boocha can you please enable logs recording in OM settings, restart OM, reproduce the issue, and send us logs via "Report a bug"?

After that, please try to type `?gl` in the search box of OM, restart OM, and let us know if it works. You can switch back to the current renderer by typing `?vulkan` and restarting the app.

**Comment by vng** [comment]
Vulkan is already disabled for "PowerVR Rogue G6110"

**Comment by boocha** [comment]
> Vulkan is already disabled for "PowerVR Rogue G6110"

Yep! Vulkan is already disable for my device. Before that I have troubles with a black screen on app startup. Now application is work well. But I have this small bug when 3D-buildings mode enabled.
